nixos/modules/services/networking/nomad.nix

```diff
@@ -148,7 +159,8 @@ in
                 };
             in
             "${cfg.package}/bin/nomad agent -config=/etc/nomad.json -plugin-dir=${pluginsDir}/bin" +
-            concatMapStrings (path: " -config=${path}") cfg.extraSettingsPaths;
+            concatMapStrings (path: " -config=${path}") cfg.extraSettingsPaths +
+            concatMapStrings (path: " -config=\${CREDENTIALS_DIRECTORY}/${lib.last (lib.splitString "/" path)}") (lib.attrValues cfg.credentials);
```

Comment:
The name of the loaded credential is `${key}`, not the last component of the origin path.

Reply:
thanks, I see.